src/extensions/Extension.ts

```diff
@@ -22,6 +22,8 @@ export async function extendAll(): Promise<void>
 		} = require(`./${file}`);
 		/* eslint-enable */
 
+		if (!Extension || !Target) continue;
```

Comment:
When would it possible to have no extension?

Reply:
When using discord.js' API for this, i.e. what changed in this PR